File: One Task/Task 1/c programming/43.c
// Transpose  of a Matrix

#include <stdio.h>

int main() {

    int m[3][3] = {
        {1, 2, 2},
        {3, 0, 4},
        {4, 1, 2}
    };
    
    int ans[3][3];

    for(int i=0; i<3; i++) {
        for(int j=0; j<3; j++) {
            ans[i][j] = m[j][i];
            printf("%2d ", ans[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Without Node Module/Xampp/htdocs/core/week-1/c-programs/programs/43.php
<?php
    $matrix = [[1, 2, 3], [4, 5, 6], [7, 8, 9]];
    $transposedMatrix = [];

    for ($i = 0; $i < count($matrix); $i++) {
        for ($j = 0; $j < count($matrix[$i]); $j++) {
            $transposedMatrix[$j][$i] = $matrix[$i][$j];
        }
    }

    echo "Transpose of Matrix:<br>";
    printMatrix($transposedMatrix);

    function printMatrix($matrix) {
        foreach ($matrix as $row) {
            echo implode(", ", $row) . "<br>";
        }
    }
?>
